applications/nrf5340_audio/README.rst

```diff
@@ -1166,8 +1169,8 @@ After building the files for the development kit you want to program, complete t
 
 .. _nrf53_audio_app_testing:
 
-Testing
-=======
+Testing the application
+=======================
```

Comment:
```suggestion
Testing
=======
```
This is always called "Testing", per template.

Reply:
Changed back to "Testing"